libmscore/style.cpp

```diff
@@ -269,10 +269,10 @@ static const StyleType styleTypes[] {
       { Sid::harmonyPlacement,         "harmonyPlacement",       int(Placement::ABOVE) },
 
       { Sid::chordSymbolAPosAbove,      "chordSymbolPosAbove",       QPointF(.0, -2.5) },
-      { Sid::chordSymbolAPosBelow,      "chordSymbolPosBelow",       QPointF(.0, 3.5) },
+      { Sid::chordSymbolAPosBelow,      "chordSymbolPosBelow",       QPointF(.0, 7.0) },
 
       { Sid::chordSymbolBPosAbove,      "chordSymbolBPosAbove",      QPointF(.0, -5.0) },
-      { Sid::chordSymbolBPosBelow,      "chordSymbolBPosBelow",      QPointF(.0, 3.5) },
+      { Sid::chordSymbolBPosBelow,      "chordSymbolBPosBelow",      QPointF(.0, 7.0) },
```

Comment:
but then this should be 2.5 below `Sid::chordSymbolAPosBelow`, shouldn't it?

Reply:
Oh yeah you're right, I will make that change once we decide what's a good offset for position below.

Reply:
I think those B types should be farther away from staff, so above A  type above but below A type below

Reply:
but if the same value for either results in that, like die to autoplace, no change needed, then actually one could use A above and B below without getting a huge gap?

Reply:
Oh yeah that makes sense. Originally, the position below for both A and B were the same so that's why I kept it. I think that's a good reason why.